data/campaigns/Heir_To_The_Throne/scenarios/08_The_Princess_of_Wesnoth.cfg

```diff
@@ -321,11 +321,16 @@
 
     [event]
         name=moveto
-        [filter]
+        [filter_condition]
```

Comment:
The `side=1` no longer has any effect now that it's in `[filter_condition]` rather than `[filter]`.